src/ape/api/providers.py

```diff
@@ -316,12 +316,19 @@ def send_call(self, txn: TransactionAPI, **kwargs) -> bytes:  # Return value of
         """
 
     @abstractmethod
-    def get_receipt(self, txn_hash: str) -> ReceiptAPI:
+    def get_receipt(
+        self, txn_hash: str, required_confirmations: int = 0, timeout: Optional[int] = None
```

Comment:
Do we picture having more kwargs here in the future? do we want to add `**kwargs` so it's not breaking?

Reply:
I think that is a good thing to do. This may save us an breaking change item in the future. Are there any other methods where a safety kwargs could be added perhaps? `send_transaction()` ?

Reply:
If we do that, I assume the other updated methods will be on separate PRs/tickets?

To recap, we want @dtdang to update this PR with **kwargs?
```suggestion
        self, txn_hash: str, required_confirmations: int = 0, timeout: Optional[int] = None, **kwargs
```